app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file_using_cp_command(command: str) -> None:
+    command_parsed = command.split(" ")
+    src_file_name = command_parsed[1]
+    dst_file_name = command_parsed[2]
+
+    if src_file_name == dst_file_name:
+        print(f"File wasn't copied: source file '{src_file_name}' "
+              f"has the same name as the destination file.")
```

Comment:
This check for the source and destination files being the same is a good idea, but it's not enough. You should also check if the source file actually exists before trying to open it. Use the os.path.isfile() function to check if a file exists.